app/src/main/java/sdp/moneyrun/EndGameActivity.java

```diff
@@ -1,6 +1,9 @@
 package sdp.moneyrun;
```

Comment:
Also a suggestion, the functionality right now includes displaying the score and a button to go back to main, maybe we would like to restart a game without needing to go to main menu and rejoin a game. If all the players want to play the "same" game again they should have an option to do so, but this could be a task to follow and to ponder if it makes sense.